scripts/deploy_kd

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -e
+
+# check manually with the released docker artifact to see the version of kd and kubectl
+# docker run --rm -ti --entrypoint=bash -v $PWD/kd:/kd -e ENVIRONMENT -e APP quay.io/ukhomeofficedigital/kd:${KD_VERSION}
+
+VERSION=$1
+KD_BINARY='kd_linux_amd64'
+KD_VERSION='v0.1.0'
+KD_MD5='45aeb54e3eb78cadfd824c1650b427ce'
+KD_DOWLOAD_ADDR="https://github.com/UKHomeOffice/kd/releases/download/${KD_VERSION}/kd_linux_amd64"
+KUBECTL_BINARY='kubectl'
+KUBECTL_VERSION='v1.3.5'
+KUBECTL_MD5='26852c9bd750a2afee527658944397d1'
+KUBECTL_DOWNLOAD_ADDR="https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl"
+ENVIRONMENT='dev'
+APP='pttg-ip-fm-ui'
+
+function getKd() {
+echo "checking if the desired kd version exists"
```

Comment:
I thought I did have indentation, not sure how it got reverted ... 
I'll indent those 2 functions.